test/suite/test_cursor13.py

```diff
@@ -465,25 +468,32 @@ def test_cursor_sweep(self):
         begin_sweep_stats = self.sweep_stats()
         #self.tty('stats before = ' + str(begin_stats))
         #self.tty('sweep stats before = ' + str(begin_sweep_stats))
+        potential_dead = 0
 
         for round_cnt in range(0, self.rounds):
             if round_cnt % 2 == 1:
                 # Close cursors in half of the range, and don't
                 # use them during this round, so they will be
                 # closed by sweep.
                 half = self.nuris / 2
-                self.close_uris(uri_map, xrange(0, half))
+                potential_dead += self.close_uris(uri_map, xrange(0, half))
                 bottom_range = half
                 # Let the dhandle sweep run and find the closed cursors.
                 time.sleep(3.0)
             else:
                 bottom_range = 0
 
+            # The session cursor sweep runs at most once a second and
+            # traverses a fraction of the cached cursors.  We'll run for
+            # ten seconds with pauses to make sure we see sweep activity.
+            pause_point = self.opens_per_round / 100
```

Comment:
While I realize `opens_per_round` is large, we probably should make sure `pause_point` does not end up as 0 to avoid a divide-by-zero later.

Reply:
Thanks, I'll add that.